### PR TITLE
Stylecop fixes Polyline3D

### PR DESCRIPTION
**Pull request by AQuentin2033**
General comments and cleanup

deleted not implemented method - IsPlanarWithinTol

### CODE REVIEW

File: src/Spatial/Euclidean/PolyLine3D.cs [hunk 1]
```diff
@@ -10,58 +10,53 @@
...
+        /// </summary>
         private readonly List<Point3D> points;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PolyLine3D"/> class.
+        /// Creates a PolyLine3D from a pre-existing IEnumerable of Point3Ds
+        /// </summary>
+        /// <param name="points">A list of points.</param>
         public PolyLine3D(IEnumerable<Point3D> points)
         {
```
Comment (JohanLarsson) [inline src/Spatial/Euclidean/PolyLine3D.cs]: We could add a null check here, throwexpression perhaps. Just a reminder, no need to fix in this PR.

File: src/Spatial/Euclidean/PolyLine3D.cs [hunk 2]
```diff
@@ -10,58 +10,53 @@
...
+        /// Gets the length of the polyline, computed as the sum of the lengths of every segment
         /// </summary>
         public double Length => this.GetPolyLineLength();
 
         /// <summary>
-        /// Indicates whether or not the collection of points in the polyline are planar within
+        /// Gets a value indicating whether or not the collection of points in the polyline are planar within
         /// the floating point tolerance
         /// </summary>
         public bool IsPlanar
```
Comment (JohanLarsson) [inline src/Spatial/Euclidean/PolyLine3D.cs]: We should remove this or write an implementaion, I lean towards removing it.

File: src/Spatial/Euclidean/PolyLine3D.cs (hunk 1 above)
Comment (AQuentin2033) [reply src/Spatial/Euclidean/PolyLine3D.cs]: this is just the doco pr, haven't yet started tuning the class

File: src/Spatial/Euclidean/PolyLine3D.cs [hunk 3]
```diff
@@ -133,20 +128,31 @@ public Point3D ClosestPointTo(Point3D p)
             return closest;
         }
 
-        public bool IsPlanarWithinTol(double tolerance)
```
Comment (JohanLarsson) [inline src/Spatial/Euclidean/PolyLine3D.cs]: Was this unused & removed?

**Pull request merged**